scripts/gulp/plugins/util/getShorthandInfo.ts

```diff
@@ -81,6 +94,35 @@ const getShorthandInfo = (componentFile: t.File, componentName: string): Shortha
         }
       }
     },
+    VariableDeclarator(path) {
+      if (path.get('id').isIdentifier({ name: componentName })) {
+        if (isCompose(path.get('init'))) {
+          const composeConfigPath = path.get('init.arguments.1') as NodePath<t.ObjectExpression>;
+          const composePropertiesPaths = composeConfigPath.get('properties');
+
+          const shorthandConfigPath = composePropertiesPaths.find(
+            property =>
+              property.isObjectProperty() &&
+              (property.get('key') as NodePath<t.Identifier>).isIdentifier({ name: 'shorthandConfig' }),
+          ) as NodePath<t.ObjectProperty> | undefined;
+
+          if (shorthandConfigPath) {
+            const shorthandConfigPropertiesPaths = shorthandConfigPath.get('value.properties') as NodePath<
+              t.ObjectProperty
+            >[];
+            const shorthandMappedPropertyPath = shorthandConfigPropertiesPaths.find(property =>
+              (property.get('key') as NodePath<t.Identifier>).isIdentifier({ name: 'mappedProp' }),
+            );
+
+            if (shorthandMappedPropertyPath) {
+              // @ts-ignore
+              t.assertStringLiteral(shorthandMappedPropertyPath.node.value);
+              mappedShorthandProp = (shorthandMappedPropertyPath.node.value as t.StringLiteral).value;
+            }
+          }
+        }
+      }
+    },
```

Comment:
Rebase issue?

Reply:
Closing PR since we not gonna focus in IE for now :)